lib/Server/LogoutHandlers/BackChannelLogoutHandler.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace SimpleSAML\Module\oidc\Server\LogoutHandlers;
+
+use Generator;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Pool;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use League\OAuth2\Server\Exception\OAuthServerException;
+use SimpleSAML\Module\oidc\Server\Associations\RelyingPartyAssociation;
+use SimpleSAML\Module\oidc\Services\LoggerService;
+use SimpleSAML\Module\oidc\Services\LogoutTokenBuilder;
+use Throwable;
+
+class BackChannelLogoutHandler
+{
+    protected LogoutTokenBuilder $logoutTokenBuilder;
+
+    protected LoggerService $loggerService;
+
+    public function __construct(
+        ?LogoutTokenBuilder $logoutTokenBuilder = null,
+        ?LoggerService $loggerService = null
+    ) {
+        $this->logoutTokenBuilder = $logoutTokenBuilder ?? new LogoutTokenBuilder();
```

Comment:
Similar to other places : I'm not sure why the object isn't instantiated in the Container and with required constructor args.

Reply:
Check comment.